tests/helpers/url.js

```diff
@@ -1,4 +1,8 @@
 const host = process.env.UNLOCK_HOST || '127.0.0.1'
 const port = process.env.UNLOCK_PORT || 3000
+const paywall = process.env.PAYWALL_URL || '127.0.0.1:3001'
 
-module.exports = (path = '/') => `http://${host}:${port}${path}`
+module.exports = {
+  main: (path = '/') => `http://${host}:${port}${path}`,
+  paywall: (path = '/') => `${paywall}/${path}`,
```

Comment:
There is something fishy here. If the `PAYWALL_URL` is not set, then that will build `127.0.0.1:3001/` which I am not sure is correct?

Reply:
this is correct. If we are running the integration tests locally outside of docker, then the PAYWALL_URL is http://127.0.0.1:3001

Reply:
Well clearly that line above `const paywall = process.env.PAYWALL_URL || '127.0.0.1:3001'`
if `PAYWALL_URL` happens to be falsy (unset), then we end up with `127.0.0.1:3001/` which will not work...